FAQ-Improvements.md

```diff
@@ -0,0 +1,27 @@
+This is a list of "realworld" notes that have been encountered... so people
+can look at "cool new feature" and find what's known/been done before.
+
+1. "Let's seal it and use inductive charging!"
+  a)The Qi chargers are larger than the PETling so they have to be bent.
+		some Qi chargers get very unhappy about that (overheating/fire),
+		while others work but are *very* picky about placement on the Qi
+		charging pad.
+    There are inductive chargers that fit inside the cap (eg: 
```

Comment:
it has been tested and it technically does work kind of, but it is anything but clean rather hacky. So that statement is not totally correct